tendermint/src/lite/verifier.rs

```diff
@@ -136,7 +147,7 @@ where
 pub fn verify_commit_trusting<V, C, L>(
     validators: &V,
     commit: &C,
-    trust_level: L,
+    trust_level: &L,
```

Comment:
It has to be a reference now because we use it several times below (and hence "moved" it). Alternative would be: use the `Copy` trait or `Clone` it explicitly.

Reply:
I would probably lean in favour of Copy or Clone since it's a small type and we dereference it to use it below, so would be more straight forward to not pass in the reference. Unless there's an argument for consistency in function parameters.